lib/certs.js

```diff
@@ -0,0 +1,95 @@
+/*jslint node: true */
+'use strict';
+
+const os = require('os');
+const path = require('path');
+const fs = require('fs-extra');
+const logger = require('@blackbaud/skyux-logger');
+const npmInstall = require('./npm-install');
+const cloneUtil = require('./utils/clone');
+
+function getCertRootPath() {
+  return path.resolve(`${os.homedir()}/.skyux/certs/`);
```

Comment:
Any reason this value can't be a `const`?

Reply:
Reworked and this is now just a const.